tempest/api/workloadmgr/snapshot/test_snapshot_mount.py

```diff
@@ -93,3 +93,137 @@ def test_1_filesearch_default_parameters(self):
             LOG.error("Exception: " + str(e))
             reporting.set_test_script_status(tvaultconf.FAIL)
             reporting.test_case_to_write()
+
+
+    @test.attr(type='smoke')
+    @test.idempotent_id('28fd0710-ef00-42b0-93f0-9dbb3ffc5bee')
+    def test_2_umount_snapshot(self):
+        reporting.add_test_script(str(__name__) + "_umount_snapshot")
+        try:
+            if self.exception != "":
```

Comment:
Do we require this if condition?? As this test case is not followed by any prerequisite call.